docs/start/getting-started/nextsteps.md

```diff
@@ -5,14 +5,15 @@ description: Getting Started with Amplify Framework - Next steps
 <inline-fragment integration="angular" src="~/start/getting-started/fragments/common/nextsteps-header.md"></inline-fragment>
 <inline-fragment integration="js" src="~/start/getting-started/fragments/common/nextsteps-header.md"></inline-fragment>
 <inline-fragment integration="vue" src="~/start/getting-started/fragments/common/nextsteps-header.md"></inline-fragment>
+<inline-fragment integration="next" src="~/start/getting-started/fragments/common/nextsteps-header.md"></inline-fragment>
 <inline-fragment integration="react" src="~/start/getting-started/fragments/common/nextsteps-header.md"></inline-fragment>
 <inline-fragment integration="react-native" src="~/start/getting-started/fragments/common/nextsteps-header.md"></inline-fragment>
 <inline-fragment integration="ionic" src="~/start/getting-started/fragments/common/nextsteps-header.md"></inline-fragment>
 
 <inline-fragment integration="angular" src="~/start/getting-started/fragments/angular/nextsteps.md"></inline-fragment>
 <inline-fragment integration="js" src="~/start/getting-started/fragments/vanillajs/nextsteps.md"></inline-fragment>
 <inline-fragment integration="vue" src="~/start/getting-started/fragments/vue/nextsteps.md"></inline-fragment>
-<inline-fragment integration="react" src="~/start/getting-started/fragments/react/nextsteps.md"></inline-fragment>
+<inline-fragment integration="next" src="~/start/getting-started/fragments/react/nextsteps.md"></inline-fragment>
```

Comment:
Is this a typo? Don't we need both `next` and `react` fragments?

Reply:
We do. The next steps are the same as other React apps, so I opted not to duplicate this fragment.

We don't have data or examples on if or when customers would want analtyics/storage/xr on the server, so I chose not have any callouts for that explicitly.